tekton/utils/pipeline.go

```diff
@@ -48,10 +48,17 @@ type Pipeline struct {
 	// +optional
 	ServiceAccount string `json:"serviceAccountName,omitempty"`
 
-	// Timeout is value to use to override the tekton default Pipelinerun timeout
+	// Timeout is a value to use to override the tekton default Pipelinerun timeout
+	//
+	// This field is DEPRECATED and will be replaced by Timeouts in a future change.
```

Comment:
Timeouts is added in this change, not a future change. Why are we keeping Timeout?

Reply:
Because this PR only adds the function but is not used. Timeout is still used here.